scene/main/node.h

```diff
@@ -423,6 +423,7 @@ class Node : public Object {
 	void add_child(Node *p_child, bool p_force_readable_name = false, InternalMode p_internal = INTERNAL_MODE_DISABLED);
 	void add_sibling(Node *p_sibling, bool p_force_readable_name = false);
 	void remove_child(Node *p_child);
+	void clear_children(bool p_include_internal = false);
```

Comment:
All other methods with an include_internal option default to true in the core. Might make sense to have that consistent.